src/ft2font.cpp

```diff
@@ -43,9 +43,23 @@
 
 FT_Library _ft2Library;
 
+// FreeType error codes; loaded as per fterror.h (modified to use std::map).
```

Comment:
I don't see any `std::map`. Is this still correct?

Reply:
Oops, that was some old implementation, good catch, fixed.